doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -281,7 +281,7 @@ The following is an example of a record with non-trivial subtyping relation:
 
 .. math::
 
-   E[Γ] ⊢ \mathsf{packType}@\{i\} =_{βδιζη}
+   E[Γ] ⊢ \mathsf{packType}@\{i\} ≤_{βδιζη}
```

Comment:
Maybe the same should be done for irrelevant universe above, but at least it's not incoherent, it's just stronger to state convertibility rather than subtyping. I'm taking about:

```diff
-E[Γ] ⊢ \mathsf{list}@\{i\}~A =_{βδιζη} \mathsf{list}@\{j\}~B
+E[Γ] ⊢ \mathsf{list}@\{i\}~A ≤_{βδιζη} \mathsf{list}@\{j\}~B
```

If yes, then the text should probably also be changed since it's explicit about the terms being convertible and not simply related by subtyping (which is the relation that matters for conversion).

Reply:
being convertible also matters, eg for `F A <= F B` you need `A = B` not `A <= B`